### DISCUSSION TITLE
Code base repo

### DISCUSSION THREAD

**Discussion by BroProgramerWeb3** [Cyfrin/foundry-full-course-cu#1976] [General]
How can i get into the code repo? because i could not find it anywhere

**Comment by EngrPips** [comment]
Hello, @KenLearnsCoding. Each lesson has its own repo, so which lesson do you need its repo?

**Reply by BroProgramerWeb3** [reply]
Thank you, i tried to reach it out by selecting the code section in the main repo.

**Reply by EngrPips** [reply]
Awesome. I am glad you figured that out.
